client/components/global/BookDetail.jsx

```diff
@@ -60,7 +28,7 @@ export default function BookDetail({
             <Image src={book.imageURL} fluid rounded />
           </Col>
           <Col>
-            <div>by {book.authors.join(', ')}</div>
+            <div>by {Array.isArray(book.authors) ? book.authors.join(', ') : book.authors}</div>
```

Comment:
google always returns an array. should not be a problem anymore once my change gets merged. significantly changes bookdetail